cytotable/utils.py

```diff
@@ -389,3 +389,28 @@ def _arrow_type_cast_if_specified(
 
     # else we retain the existing data field type
     return column
+
+
+def _expand_path(
+    path: Union[str, pathlib.Path, CloudPath]
+) -> Union[str, pathlib.Path, CloudPath]:
+    """
+    Expands provided path with os environment or user expanded paths.
+
+    Args:
+        path: Union[str, pathlib.Path, CloudPath]:
+            Path to expand.
+
+    Returns:
+        Union[pathlib.Path, Any]
```

Comment:
Thanks - good point, I think this was initially done to avoid type-hint linting discrepancies experienced through Mypy. On testing this more lately there doesn't appear to be an issue in moving to AnyPath. I've updated things accordingly with another commit.